pricegraph/src/orderbook/scalar.rs

```diff
@@ -169,3 +167,122 @@ fn assert_strictly_positive_and_finite(value: f64) -> f64 {
     debug_assert!(num::is_strictly_positive_and_finite(value));
     value
 }
+
+/// A signed fixed point number with 24 magnitude bits and 104 fractional bits.
```

Comment:
```suggestion
/// A signed fixed point number with 1 sign bit, 23 magnitude bits and 104 fractional bits.
```
Maybe? Not sure if this is usually counted towards the magnitude or not.

Reply:
AFAIK, fixed point numbers typically use 2's complement notation so that they reuse integer math.